### PR TITLE
Ignore null property types when generating RDF.

### PR DESCRIPTION
**Pull request by elshimone**
This fixes an issue I found after applying the changes to make QuantityValue bounds optional (https://github.com/Wikidata/Wikidata-Toolkit/pull/247).

Exporting a wikidata dump as rdf triggered a null pointer exception:

Exception in thread "main" java.lang.NullPointerException
        at org.wikidata.wdtk.rdf.SnakRdfConverter.getRangeUri(SnakRdfConverter.java:307)

It looks like the intention of the code was to return null in the case of an unknown property type, this informed my fix.

### CODE REVIEW

File: wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/SnakRdfConverter.java [hunk 1]
```diff
@@ -304,6 +304,9 @@ String getRangeUri(PropertyIdValue propertyIdValue) {
 		String datatype = this.propertyRegister
 				.getPropertyType(propertyIdValue);
 
+		if (datatype == null)
+			return null;
```
Comment (thiemowmde) [inline wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/SnakRdfConverter.java]: I believe the `@return` above should be adjusted.

File: wdtk-rdf/src/test/java/org/wikidata/wdtk/rdf/MockPropertyRegister.java [hunk 2]
```diff
@@ -47,6 +47,12 @@ public MockPropertyRegister() {
 
 	}
 
+	public MockPropertyRegister (String uriPatternPropertyId,
+								 ApiConnection apiConnection, String siteUri) {
+		super(uriPatternPropertyId, apiConnection, siteUri);
```
Comment (thiemowmde) [inline wdtk-rdf/src/test/java/org/wikidata/wdtk/rdf/MockPropertyRegister.java]: Why is this necessary if it doesn't change the signature anyway?

**Comment by guenthermi** [comment]
Thank you! I just removed the second constructor and merged the Code via #249.
